client/src/utils/apiFunctions.js

```diff
@@ -37,9 +37,21 @@ export const logoutUser = () => {
 };
 
 export const getInterview = (roomId) => {
-  return api.get(`/interviews/${roomId}`)
+  return api.get(`/interviews/${roomId}`);
 };
 
 export const getQuestion = (questionId) => {
-  return api.get(`/questions/${questionId}`)
+  return api.get(`/questions/${questionId}`);
+};
+
+export const createFeedback = (interviewId, formData) => {
+  return api.put(`/interviews/${interviewId}/feedback/`, formData);
+};
+
+export const getFeedbackCreator = (interviewId) => {
+  return api.get(`/interviews/${interviewId}/feedback/creator`);
```

Comment:
Is creator in the function name and route because it's the feedback you've created vs the feedback you've received?